mitty/test/data/gt_free.vcf

```diff
@@ -0,0 +1,115 @@
+##fileformat=VCFv4.1
+##FILTER=<ID=PASS,Description="All filters passed">
+##contig=<ID=1>
+#CHROM	POS	ID	REF	ALT	QUAL	FILTER	INFO
+1	10146	.	AC	A	10477.8	.	.
```

Comment:
Would it be better to have  a smaller test file with `AF` values so you can explicitly test the algorithm?